src/osd/scrubber/osd_scrub_sched.cc

```diff
@@ -204,10 +204,10 @@ void ScrubQueue::register_with_osd(ScrubJobRef scrub_job,
       break;
   }
 
-  dout(10) << "pg(" << scrub_job->pgid << ") sched-state changed from "
-	   << qu_state_text(state_at_entry) << " to "
-	   << qu_state_text(scrub_job->state)
-	   << " at: " << scrub_job->schedule.scheduled_at << dendl;
+  dout(10) << fmt::format(
+		"pg[{}] sched-state changed from <{:.14}> to <{:.14}>",
+		scrub_job->pgid, state_at_entry, scrub_job->state.load())
+	   << dendl;
```

Comment:
Removal of `scrub_job->schedule.scheduled_at` is intentional here? Just checking.

Reply:
don't think so... restored. Thanks!